ui/widgets/nuxeo-html-editor.js

```diff
@@ -182,7 +182,7 @@ import './quill/quill-snow.js';
     _valueChanged() {
       if (this._editor && !this._internalChange) {
         const delta = this._editor.clipboard.convert({ html: this.value });
-        this._editor.setContents(delta, Quill.sources.USER);
+        this._editor.setContents(delta, this.readOnly ? Quill.sources.SILENT : Quill.sources.USER);
```

Comment:
We know that if the value is changing and we are in read only mode, it cannot have been triggered by user, so changing the source API seemed like a clean solution. An alternative could be to add a couple of ifs checking if we are in read only mode and then temporarily re-enabling the editor, but it seemed for fiddly to me. WDYT?

Reply:
This one looks clean enough like this 👍